test/parallel/test-sync-io-option.js

```diff
@@ -20,7 +20,7 @@ if (process.argv[2] === 'child') {
     execFile(process.execPath, args, function(err, stdout, stderr) {
       assert.equal(err, null);
       assert.equal(stdout, '');
-      if (/WARNING[\s\S]*fs\.readFileSync/.test(stderr))
+      if (/WARNING[\s\S]*readFileSync/.test(stderr))
```

Comment:
Naming anonymous functions produces a stacktrace with just the function name now. For example, earlier it used to be `fs.readFileSync` and now it has become just `readFileSync` in the stacktrace.

Is that okay @nodejs/collaborators ?